libminifi/include/utils/HTTPUtils.h

```diff
@@ -88,6 +90,40 @@ struct HTTPRequestResponse {
 
 };
 
+static void parse_url(std::string &url, std::string &host, int &port, std::string &protocol) {
```

Comment:
for now, we can document the same in the README.md to specify the URL format if it is OK with you. 
As for the access control, i looked at the doc, it looks like we need to use token, etc for a secure cluster. I will need to ask around.